jrnl/cli.py

```diff
@@ -313,6 +314,14 @@ def configure_logger(debug=False):
 
 
 def run(manual_args=None):
+    if packaging.version.parse(platform.python_version()) < packaging.version.parse(
+        "3.7"
+    ):
+        print(
+            "WARNING: Python versions below 3.7 are not supported by jrnl\n",
```

Comment:
Can we make this a bit friendlier, and add color?

What do you think of something like this?

```
f"""{WARNING_COLOR}
WARNING: Python versions below 3.7 will no longer be supported as of jrnl v2.5 (the next
release). You are currently are on Python {platform.python_version()}. Please update to
Python 3.7 (or higher) soon.
{RESET_COLOR}"""
```